Controllers/BarcodeController.cs

```diff
@@ -46,4 +50,17 @@ public async Task<IActionResult> DeleteBarcode(string barcodeName)
         await _dbInfo.Barcodes.Where(x => x.Key == barcodeName).ExecuteDeleteAsync();
         return Ok();
     }
+
+    [HttpGet]
+    [Route($"{_controllerRoute}/suggest")]
+    public async Task<IActionResult> SuggestBarcode(string barcodeName) 
+    {
+        return Ok( 
+            await _dbInfo.Barcodes
+            .Where(x => x.Key == barcodeName)
```

Comment:
czy nie chciałeś tutaj bardziej coś typu where(x => x.Key.Contains(barcodeName)) ?
to sie rozwinie w SQL do "where key like '%param%'"

Reply:
Na razie zostawie to, jak nie będzie działać to zmienię xd